schemas/public/ekscluster-kfd-v1alpha2.json

```diff
@@ -1452,6 +1452,10 @@
         "deadManSwitchWebhookUrl": {
           "type": "string"
         },
+        "defaultRules":  {
+          "type": "string",
+          "enum": ["enabled", "disabled"]
```

Comment:
Same as above, i would like the file to be installDefaultRules and a boolean type

Reply:
I got an error when on "reducer" when try to put a boolean